module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -182,6 +182,23 @@ class KohaRest extends \VuFind\ILS\Driver\AbstractBase implements
      */
     protected $itemStatusMappings = [];
 
+    /**
+     * Item status mapping methods used when the item status mappings above don't
```

Comment:
If methods are only used when mappings are undefined, why not set mappings instead of defining methods for the situations where a hard-coded value is returned without any logic? Or am I misunderstanding?

Reply:
They can be used at the same time. You can e.g. configure a single status differently just by adding it to the mappings in the .ini file. Methods are used for any statuses that don't have a mapping.

Reply:
Understood, but my question is, if the method is simply returning a hard-coded string value, would it make more sense to make the default implementation a default entry in the mappings array instead of a custom method? If somebody wanted to override with a custom method in a subclass, they could still null out the mappings entry and add a method map entry. Or am I missing a nuance that makes this impractical/inappropriate?

Reply:
Sorry, I misunderstood. Totally makes sense. Now simplified.